components/ui/Switch.tsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from "react";
+import * as PrimitiveSwitch from "@radix-ui/react-switch";
+import * as Label from "@radix-ui/react-label";
+
+function classNames(...classes) {
+  return classes.filter(Boolean).join(" ");
+}
+
+export default function Switch(props) {
+  const { label, ...primitiveProps } = props;
+  const [checked, setChecked] = useState(false);
+
+  return (
+    <div className="flex items-center h-[20px]">
```

Comment:
would this maybe cause issues if it's fixed 20px? haven't looked into the visuals yet